### PR TITLE
Expose .lu and .qna as static files for skill scenario

### PR DESCRIPTION
**Pull request by luhan2017**
### Purpose
*What is the context of this pull request? Why is it being done?*

This PR is to expose .lu and .qna as static files for skill scenarios. 

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by gabog — COMMENTED** [review]
I would defer to @peterinnesmsft and @tracyboehrer to approve and merge this but it looks OK to me.
We should also add similar functionality to the JS runtime.

**Review by tracyboehrer — APPROVED** [review]

**Comment by tracyboehrer** [comment]
Wait for Peter to merge.

**Review by peterinnesmsft — APPROVED** [review]

**Pull request merged**
